algorithms/CPlusPlus/Arrays/watering-plants-leetcode.cpp

```diff
@@ -0,0 +1,36 @@
+#include<iostream>
+#include<bits/stdc++.h>
+using namespace std;
+
```

Comment:
Provide some description for the program

Reply:
Have added the problem statement